subsys/bluetooth/host/Kconfig.gatt

```diff
@@ -142,6 +142,14 @@ config BT_GATT_READ_MULTIPLE
 	  This option enables support for the GATT Read Multiple Characteristic
 	  Values procedure.
 
+config BT_GATT_READ_MULTIPLE_VARIABLE_LENGTH
+	bool "GATT Read Multiple Variable Length Characteristic Values support"
+	default y
+	depends on BT_GATT_READ_MULTIPLE
```

Comment:
Hmm, according the table in the core spec, EATT mandates `Read Multiple Variable Length Characteristic Values` but `Read Multiple Characteristic Values` is optional, implying that we can support `Read Multiple Variable Length Characteristic Values` without `Read Multiple Characteristic Values`

Reply:
Ok, will remove the `depends on` here and move some ifdefs on `CONFIG_BT_READ_MULTIPLE`.